Automattic/vip-go-mu-plugins
# Revert "Disabling WebP images upload temporarily"

## Description

WordPress 5.8 [added support for WebP images](https://make.wordpress.org/core/2021/06/07/wordpress-5-8-adds-webp-support/). Previously, we disabled WebP uploads due to certain VIP Files Service issues. The issues are resolved, so we're re-enabling WebP uploads.

Reverts Automattic/vip-go-mu-plugins#2299 

## Changelog Description

### Enabling WebP uploads

We have enabled WebP uploads in WordPress Dashboard.

All WebP files would be automatically converted in JPEG for clients not supporting WebP.

jpeg and png files will continue to be converted to and served as webp files to clients that request them. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Go to `wp-admin` > `Media Library`
3. Try to upload a WebP image. The image should be successfully uploaded.
